tests/config_end_to_end_test.yml

```diff
@@ -0,0 +1,23 @@
+experiment_name: end_to_end_test
+n_repeats: 1
+scenario_params_list:
+ - dataset_name:
+    'mnist':
+     - 'random_initialization'
+   partners_count:
+     - 3
+   amounts_per_partner: 
+     - [0.4, 0.3, 0.3]
+   samples_split_option: 
+     - ['basic', 'random']
+   multi_partner_learning_approach:
+     - 'fedavg'
+     - 'seq-pure'
+   epoch_count: 
+     - 1
+   minibatch_count: 
+     - 10
+   gradient_updates_per_pass_count:
+     - 8
+   dataset_proportion:
+     - 1
```

Comment:
That is a lot of sample... shouldn't we keep it to `0.1` or so? (#greenfuture :muscle: )

Reply:
Yes, good idea !